store/engine/consul/consul.go

```diff
@@ -312,3 +316,10 @@ func (c *Consul) Close() error {
 	c.client = nil
 	return nil
 }
+
+func sanitizeKey(key string) string {
+	if len(key) > 0 && key[0] == '/' {
+		key = strings.TrimPrefix(key, "/")
+	}
```

Comment:
trimming the leading `/` to avoid the error
https://github.com/hashicorp/consul/blob/main/api/kv.go#L208-L210